cornflow/tests/unit/test_instances.py

```diff
@@ -0,0 +1,79 @@
+from cornflow.models import InstanceModel
+from cornflow.tests.custom_test_case import CustomTestCase
+
+INSTANCE_PATH = './cornflow/tests/data/new_instance.json'
+INSTANCES_LIST = [INSTANCE_PATH, './cornflow/tests/data/new_instance_2.json']
+
+
+class TestInstancesListEndpoint(CustomTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.url = '/instance/'
+        self.model = InstanceModel
+
+    def test_new_instance(self):
+        self.create_new_row(INSTANCE_PATH)
+
+    def test_get_instances(self):
+        self.get_rows(INSTANCES_LIST)
+
+    def test_get_no_instances(self):
+        self.get_no_rows()
+
+
```

Comment:
missing (maybe add it as a TODO?);
* create an incorrect instance and validate the response. 
* create an incomplete instance.
* get a non-existent instance.
* all the above, for executions.

Reply:
I had it in mind but I am trying to have the basic test first (coverage) and then test more complex things.

Adding to that list: test with different users

Reply:
yes, indeed. do add it somewhere so we do not forget about it and anyone reading through the code can can implement it. thanks.